src/Core/Addon/Module/ModuleManager.php

```diff
@@ -752,4 +775,17 @@ private function checkConfirmationGiven($action, Module $module)
             }
         }
     }
+
+    /**
+     * Clear smarty and Symfony cache (the sf2 cache is remove on the process shutdown).
+     */
+    private function clearCache()
+    {
+        if ($this->cacheCleared) {
+            return;
+        }
+
+        Tools::clearAllCache();
```

Comment:
Maybe we should select which cache we want to clear instead of throwing everything away?

Reply:
well it's complicated, we need to clear the routing cache (stored in a generated class) and the services cache
the cache service from Symfony does not offer such precise clearing unfortunately

Reply:
I simplified it to only clear symfony cache.